src/scripts/ui/FillPlayerTradeModal.tsx

```diff
@@ -35,6 +35,12 @@ export function FillPlayerTradeModal({ tradeId, xy }: { tradeId: string; xy?: Ti
    const allTradeBuildings = Tick.current.playerTradeBuildings;
    const [fills, setFills] = useState(new Map<Tile, number>());
 
+   const DEFAULT_HINTS = "greedy";
```

Comment:
I am not sure why `DEFAULT_HINTS` is needed here. Having an opaque variable should generally be avoided